lib/s3_meta_sync/syncer.rb

```diff
@@ -83,6 +83,18 @@ def download(source, destination)
       end
     end
 
+    def mktmpdir
```

Comment:
I'm using the syntax suggested here https://ruby-doc.org/stdlib-2.3.1/libdoc/tmpdir/rdoc/Dir.html (last comment block)
would renaming the method look better?